mantidproject/mantid
# Exposes CorrectDetectors in Reflectometry GUI Settings Tab

Exposes the `CorrectDetectors` option via a checkbox on the ISIS Reflectometry GUI settings tab.

**To test:**
1. Open up the ISIS Reflectometry Interface.
2. Open the .tbl file `INTER_NR_test2.tbl` from the ISIS sample data.
3. Go to the settings tab and check the new checkbox, `CorrectDetectors`
4. Return to the Runs tab and change the angle of run `13462` from 2.3 to *3.5*
5. Select the row `13462` and press process.
6. Once processing is complete right click on the workspace `IvsQ_13462` and select `Show Detectors`
Check that the theta angle is 7 (aka 2 * 3.5).
7. Return to the settings tab and uncheck the box `CorrectDetectors` box.
8. Return to the runs tab and repeat steps 5 and 6, this time checking that the theta angle is *~4.66*.



Resolves #21337 . 

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
